.github/workflows/pr-comment.yaml

```diff
@@ -10,12 +10,11 @@ on:
         description: The ecosystem workflow that triggers the workflow run
         required: true
 
-permissions:
-  pull-requests: write
-
 jobs:
   comment:
     runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
```

Comment:
Is this change to isolate the permissions only to a single job (although we only have one in this workflow file)? I'm just trying to understand the change.

Reply:
Makes sense, thanks!